packages/logger/src/client-logger.ts

```
@@ -0,0 +1,55 @@
+    this.logLevels = ["error", "warn", "info", "debug"];
+  }
+
+  static getInstance(loggerOptions?: ILoggerOptions) {
+    if (!ClientLogger.instance) {
+      ClientLogger.instance = new ClientLogger(loggerOptions);
+    }
+    return ClientLogger.instance;
+  }
+
+  public static getLogger(loggerOptions?: ILoggerOptions): IClientLogMethods {
+    const instance = this.getInstance(loggerOptions);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)